any/Unit-Conversion

```diff
@@ -0,0 +1,41 @@
+# Unit Conversion
```

Comment:
It looks like the template is specific to a meters to feet conversion. If you want to keep it that way, then we should make the title more specific to that. 

However, I think keeping it generic is fine. See additional comments for reasons why.